waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/model/AbstractMetaStore.java

```diff
@@ -243,4 +244,12 @@ public String toString() {
         .toString();
   }
 
+  public Map<String, String> getConfigurationProperties() {
```

Comment:
Please move these two methods below the `setDatabaseNameMapping` method to keep them in sync with the property definitions. Normally, `toString` (if defined) is the last method in a class.